README.md

```diff
@@ -116,6 +116,14 @@ Use `.safetensors` files for guaranteed security.
 
 As you may know, loading `.pth` files [poses a security risk](https://github.com/pytorch/pytorch/issues/52596) due to python's `pickle` module being inherently unsafe and vulnerable to arbitrary code execution (ACE). To mitigate this, Spandrel only allows deserializing certain types of data. This helps to improve security, but it still doesn't fully solve the issue of ACE.
 
+## Used By
+
+Here are some cool projects that use Spandrel:
+
+- [chaiNNer](https://github.com/chaiNNer-org/chaiNNer)
+- [AUTOMATIC1111's SD WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
+- [dgenerate](https://github.com/Teriks/dgenerate)
```

Comment:
We should probably order them by GH stars just to put the more popular ones first.

```suggestion
- [AUTOMATIC1111's SD WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
- [chaiNNer](https://github.com/chaiNNer-org/chaiNNer)
- [dgenerate](https://github.com/Teriks/dgenerate)
```